manual/manual.adoc

```diff
@@ -167,19 +175,19 @@ messages shown when you collect an item.
 
 == How to Play
 
-image::images/c5m1-sshot.png[Freedoom Screenshot,width="640",pdfwidth="70vw",align="center"]
+image::images/map01-sshot.png[Freedoom Screenshot,width="640",pdfwidth="70vw",align="center"]
 
 Freedoom is a real-time first-person shooter (FPS). You’ll be exploring a
 series of levels, in each one trying to find the way to the exit. An
 assortment of monsters will try to stop you, and you’ll need to use weapons
 to defend yourself.
 Portions of the levels may be inaccessible until you find a particular key, or
-find a switch to open a locked door. This gives a puzzle element to the game in
+find a switch to open a passage. This gives a puzzle element to the game in
 addition to the action.
 
 By default, the keyboard cursor keys will move forward and backward, and turn
 left and right. The _Control_ key fires the current weapon, and the spacebar
-will open doors and activate switches. All source ports allow you to rebind
+will open doors and activate switches. Consult your source port on how to rebind
```

Comment:
Consult the documentation for your source port? Or should we instead recommend checking the settings for your source port?